packages/contractkit/src/governance/proposals.ts

```diff
@@ -59,8 +59,14 @@ export const proposalToJSON = async (kit: ContractKit, proposal: Proposal) => {
     if (parsedTx == null) {
       throw new Error(`Unable to parse ${tx} with block explorer`)
     }
+
+    const contractName =
```

Comment:
Why do this here vs. adjusting `blockExplorer`s `CallDetail` to include the info whether or not this really is a proxy we are calling?